src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchServiceTests.cs

```diff
@@ -34,6 +34,7 @@ public class SearchServiceTests
 
         private readonly IReadOnlyList<Tuple<string, string>> _queryParameters = new Tuple<string, string>[0];
         private readonly IReadOnlyList<Tuple<string, string>> _unsupportedQueryParameters = new Tuple<string, string>[0];
+        private readonly IReadOnlyList<(string searchParameterName, string reason)> _unsupportedSortingParameters = Array.Empty<(string searchParameterName, string reason)>();
```

Comment:
Should a test be added to show that all sort parameters are being added to unsupported sorting parameters?

Reply:
Good idea